runtime/src/accounts.rs

```diff
@@ -252,6 +252,7 @@ impl Accounts {
 
                     if solana_sdk::sysvar::instructions::check_id(key) {
                         if message.is_writable(i, demote_program_write_locks) {
+                            error_counters.invalid_account_index += 1;
```

Comment:
the message writable check will always return false for the instructions sysvar so this will never be hit anyways fyi

Reply:
True, I guess we should remove that check entirely?

Reply:
Yeah, I'm in favour

Reply:
✅